data/tutorials/getting-started/1_01_a_tour_of_ocaml.md

```diff
@@ -11,13 +11,26 @@ recommended_next_tutorials:
   - "lists"
 ---
 
-Before proceeding with this tutorial, please ensure you've installed OCaml and set up the environment, as described on the [install OCaml](/docs/installing-ocaml) page. After we take an introductory tour of OCaml's language features, we'll proceed to create our first OCaml project in the [Your First OCaml Program](/docs/your-first-program) tutorial.
+## Introduction
+Hop on the OCaml sightseeing bus.
+This absolute beginner tutorial will drive you through the marvels and wonders of OCaml.
+We'll have a look at the most commonly used language features.
+No OCaml or functional programming knowledge is required; however, it is assumed the reader has some basic software development knowledge.
 
-You need to have OCaml installed. No OCaml or functional programming knowledge is required; however, it is assumed the reader has some basic software development knowledge. This tutorial is probably not adapted to learn programming.
+The complete tour is divided in two parts.
+This first part introduces OCaml's basic features: values, expressions, lists, functions, pattern matching, and more.
+In the second part, we'll see how to create our first OCaml project in the [Your First OCaml Program](/docs/your-first-program) tutorial.
 
-This document will cover how to use the REPL UTop to evaluate OCaml expressions interactively, understand the output, how to use pattern matching, call functions from OCaml standard library modules, and more. It also introduces you to lists, values, functions, integers, floats, references, and arrays.
 
-Let's walk through the basics of OCaml by trying out different elements in an interactive manner. We recommend that you execute the examples we provide, or slight variants of them, in your own environment to get a feel for coding in OCaml.
+## Before We Begin
+Before proceeding with this tutorial, please ensure you've installed OCaml and set up the environment, as described on the [install OCaml](/docs/installing-ocaml) page.
+
+
+To walk through the basics of OCaml by trying out different elements in an interactive manner, we'll use UTop.
+UTop (Universal Toplevel) allows interacting directly with OCaml by reading and evaluating OCaml phrases like expressions or value definitions and printing the result on the screen.
+You can run UTop by using the `utop` command.
```

Comment:
```suggestion
To walk through the basics of OCaml by trying out different elements in an interactive manner, we'll use UTop. You can run UTop by using the `utop` command. Exit it by pressing `Ctrl+D`.

UTop (Universal Toplevel) allows interacting directly with OCaml by reading and evaluating OCaml phrases like expressions or value definitions and printing the result on the screen.
```

Reply:
@cuihtlauac 
Yes we need to mention how to exit UTop :-D
I don't think the shift of sentences helps us. 
This is my reasoning:
1st Sentence: Why we'll use something call UTop
2nd Sentence: What is that thing that we will use.
3rd Sentence: How you can start using it.
If we shift the sentences  I feel like it breaks a bit the dynamic.
Does this make sense?